<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Case Data Editor</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item>
    <widget class="QTableWidget" name="table_widget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
